projects/extension/pyproject.toml

```diff
@@ -8,14 +8,16 @@ dynamic = ["version"]
 requires-python = ">=3.10"
 
 dependencies= [
-    "openai==1.44.0",
+    "openai==1.56.0",
```

Comment:
why this change?

Reply:
LiteLLM requires >=1.55.3
https://github.com/BerriAI/litellm/blob/a835baacfc5e4c64690ac2dc60f2015e91222b4d/pyproject.toml#L24